Nos/Assets/Localization/Localized.swift

```diff
@@ -103,6 +103,8 @@ enum Localized: String, Localizable, CaseIterable {
     case done = "Done"
     case editProfile = "Edit Profile"
     case youNeedToEnterAPrivateKeyBeforePosting = "You need to enter a private key in Settings before you can publish posts."
+    case relayDoesNotSupportNIP40 = "This relay doesn't support expiring messages. Please select another one."
+    case anyRelaysSupportingNIP40 = "There aren't any relays that support expiring messages in your relay list. Please add one and retry."
```

Comment:
I liked Sebastian's suggestion of "None of your relays support expiring messages", do you?
```suggestion
    case anyRelaysSupportingNIP40 = "None of your relays support expiring messages. Please add one and retry."
```